src/Moq/Obsolete/MockFactory.cs

```diff
@@ -363,8 +363,18 @@ public virtual void VerifyAll()
 		}
 
 		/// <summary>
-		/// Invokes <paramref name="verifyAction"/> for each mock 
-		/// in <see cref="Mocks"/>, and accumulates the resulting 
+		/// Calls <see cref="Mock.VerifyNoOtherCalls"/> on all mocks created by this factory.
+		/// </summary>
+		/// <seealso cref="Mock.VerifyNoOtherCalls"/>
+		/// <exception cref="MockException">One or more mocks had setups that were not satisfied.</exception>
+		public virtual void VerifyNoOtherCalls()
```

Comment:
By default, methods should not be `virtual`. Do you see a reason why this method should be overridable? (If not, please make this method non-`virtual`.)